jikesrvm/rvm/src/org/jikesrvm/mm/mminterface/CollectorThread.java

```diff
@@ -77,6 +77,8 @@ public CollectorThread(byte[] stack, CollectorContext context) {
     if (!VM.BuildWithRustMMTk) {
       rvmThread.collectorContext = context;
       rvmThread.collectorContext.initCollector(nextId);
+    } else {
+      rvmThread.activeMutatorContext = false;
```

Comment:
I am not completely sure about this change though. For JikesRVM, if `activeMutatorContext` is false, that means the thread is about to die. I don't know the consequence about it. Would it be better if you just check  with `is_collector()` when getting next mutator?

Reply:
Yes, that's a better idea, I changed the implementation to use ``!is_collector()`